packages/discovery-react-components/src/components/StructuredQuery/utils/stringifyStructuredQuerySelection.ts

```diff
@@ -0,0 +1,32 @@
+import { StructuredQuerySelection } from './structuredQueryInterfaces';
+
+export const stringifyStructuredQuerySelection = (
+  structuredQuerySelection: StructuredQuerySelection
+) => {
+  let stringifiedStructuredQuerySelection: string = '';
+  stringifiedStructuredQuerySelection += Object.keys(structuredQuerySelection.groups)
+    .map(groupId => {
+      return groupId === '0'
```

Comment:
should the condition be whether or not there are multiple rows (i.e. `rows.length > 1`) before wrapping in parentheses?

Reply:
I don't think so because, at least on cloud, the initial top-level rules are never wrapped in parentheses, but then the rule(s) for all of the nested rule groups are.